core/src/main/java/com/orientechnologies/orient/core/storage/cluster/v2/OClusterPositionMapV2.java

```diff
@@ -346,7 +346,7 @@ public void remove(final long clusterPosition, final OAtomicOperation atomicOper
           if (bucket.exists(i + index)) {
             final OClusterPositionMapBucket.PositionEntry val = bucket.get(i + index);
             assert val != null;
-            result[entriesCount] = new OClusterPositionEntry(startIndex + i, val.getPageIndex(), val.getRecordPosition());
+            result[entriesCount] = new OClusterPositionEntry(index + i, val.getPageIndex(), val.getRecordPosition());
```

Comment:
Is it a fix for a different bug...?

Reply:
yes, probably i will do a differeent PR for this

Reply:
i moved this to da a separate commit